packages/sdk/src/document/document.ts

```
@@ -387,6 +398,32 @@ export interface EpochMismatchEvent extends BaseDocEvent {
   };
 }
 
+/**
+ * `LocalChangesDroppedReason` enumerates why the offline-persistence layer
+ * had to discard un-pushed local changes it could not reconcile.
+ */
+export type LocalChangesDroppedReason =
+  | 'epoch-reanchor'
+  | 'document-purged'
+  | 'actor-mismatch'
+  | 'restore-failed';
```

ESLint

[error] Replace `|·'epoch-reanchor'⏎··|·'document-purged'⏎··|·'actor-mismatch'⏎·` with `'epoch-reanchor'·|·'document-purged'·|·'actor-mismatch'`

(prettier/prettier)